labkey/domain.py

```diff
@@ -232,9 +234,11 @@ def __init__(self, **kwargs):
             "template_description", kwargs.pop("templateDescription", None)
         )
 
-        fields = kwargs.pop("fields", [])
         fields_instances = []
-
+        fields = kwargs.pop("fields", [])
+        for field in fields:
+            fields_instances.append(PropertyDescriptor(**field))
+        fields = kwargs.pop("calculatedFields", [])
```

Comment:
We should support both snake_case and camelCase here like we do for the other properties
```suggestion
        fields = kwargs.pop("calculated_fields", kwargs.pop("calculatedFields", []))
```

Reply:
good catch. done